src/components/data-exploration/generic-gene-table/SelectionActions.jsx

```diff
@@ -22,6 +23,7 @@ const SelectionActions = (props) => {
   const selectedGenes = useSelector((state) => state.genes.selected);
   const [copied, setCopied] = useState(false);
   const [listed, setListed] = useState(false);
+  const [pathAnalysisModal, setpathAnalysisModal] = useState(false);
```

Comment:
`setpathAnalysisModal` --> `setPathAnalysisModal`

also if possible avoid abbreviating: `setPathwayAnalysisModalVisible` might be better